src/main/scala/com/databricks/spark/csv/package.scala

```diff
@@ -58,6 +81,7 @@ package object csv {
                       compressionCodec: Class[_ <: CompressionCodec] = null): Unit = {
       // TODO(hossein): For nested types, we may want to perform special work
       val delimiter = parameters.getOrElse("delimiter", ",")
+      val quote = parameters.get("quote")
```

Comment:
Is there some reason this won't be a scala map? Shouldn't it return an option we can match on?

```
scala> val m: Map[String, String] = Map()
m: Map[String,String] = Map()

scala> val noString = m.get("key")
noString: Option[String] = None
```